README.md

```diff
@@ -75,7 +75,7 @@ kubectl get deployments --namespace=nginx-example
 Finally, create an alias for the Ark client's Docker executable. (Make sure that your `KUBECONFIG` environment variable is pointing at the proper config first). This will save a lot of future typing:
 
 ```
-alias ark='docker run --rm -v $(dirname $KUBECONFIG):/kubeconfig -e KUBECONFIG=/kubeconfig/$(basename $KUBECONFIG) gcr.io/heptio-images/ark:latest'
+alias ark='docker run --rm -u$(id -u) -v $(dirname $KUBECONFIG):/kubeconfig -e KUBECONFIG=/kubeconfig/$(basename $KUBECONFIG) gcr.io/heptio-images/ark:latest'
```

Comment:
nit: could you put a space here for consistency (e.g. `-u $(id -u)`)?
otherwise looks good!